include/roboteam_ai/interface/widgets/GraphWidget.h

```diff
@@ -0,0 +1,32 @@
+//
+// Created by mrlukasbos on 22-9-19.
+//
+#include <QtCharts/QtCharts>
+
+#ifndef RTT_GRAPHWIDGET_H
+#define RTT_GRAPHWIDGET_H
+
+namespace rtt {
+namespace ai {
+namespace interface {
+
+class GraphWidget : public QWidget {
+    Q_OBJECT
+private:
+    float seriesIndex = 0;
+    float fpsyMax = 0;
```

Comment:
I find the prefix fps confusing, since it's not clear what fps you're talking about (roboteam AI? the widget itself?). Maybe just use graphYMax or so?